src/task.c

```diff
@@ -737,7 +745,10 @@ DLLEXPORT void gdblookup(ptrint_t ip)
     char *func_name;
     size_t line_num;
     char *file_name;
-    frame_info_from_ip(&func_name, &line_num, &file_name, ip, 0);
+    size_t inlined_line;
+    char *inlined_file;
+    frame_info_from_ip(&func_name, &file_name, &line_num, &inlined_file, &inlined_line, ip,
+                      /* skipC */ 0, /* skipInline */ 1);
```

Comment:
it looks like we need a `free` call for `file_name` and `inlined_file` at the end of this function